### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by seanpdoyle**
Add and updates community oriented documents and guidelines.

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Sean Doyle
+Copyright (c) 2015 Sean Doyle and thoughtbot, inc.
```
Comment (mike-burns) [inline LICENSE.txt]: Woot!

You could list 2014 and 2015, or even do both like:

```
Copyright (c) 2014 Sean Doyle
Copyright (c) 2015 Sean Doyle and thoughtbot, inc.
```

File: README.md [hunk 2]
```diff
@@ -145,8 +145,41 @@ JSON::Matchers.schema_root = "docs/api/schemas"
...
+`json-matchers` was written is maintained by Sean Doyle.
+
+Many improvements and bugfixes were contributed by the [open source community].
+
+[CONTRIBUTING]: https://github.com/thoughtbot/json-matchers/blob/master/CONTRIBUTING.md
+[open source community]: https://github.com/thoughtbot/json-matchers/graphs/contributors
+
+## Need Help?
+
+We offer 1-on-1 coaching. We can help you set up a new Rails application, write
```
Comment (mike-burns) [inline README.md]: We no longer offer coaching, IIRC.

Comment (seanpdoyle) [inline README.md]: I pulled this directly from [suspenders](https://github.com/thoughtbot/suspenders#need-help).

Should we remove this line from all library READMEs?

Comment (mike-burns) [inline README.md]: Yes, though you might want to double-check in #general before doing so.

**Comment by seanpdoyle** [comment]
@mike-burns does this look good to go?

**Comment by mike-burns** [comment]
You bet'cha.

File: json-matchers.gemspec [hunk 3]
```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |spec|
   spec.name          = "json-matchers"
   spec.version       = JSON::Matchers::VERSION
   spec.authors       = ["Sean Doyle"]
-  spec.email         = ["seandoyle@thoughtbot.com"]
+  spec.email         = ["sean.p.doyle24@gmail.com"]
   spec.summary       = %q{Validate your Rails JSON API's JSON}
   spec.description   = %q{Validate your Rails JSON API's JSON}
   spec.homepage      = "https://github.com/seanpdoyle/json-matchers"
```
Comment (teoljungberg) [inline json-matchers.gemspec]: Should this be updated to point to thoughtbot?

File: LICENSE.txt [hunk 4]
```diff
@@ -1,4 +1,5 @@
 Copyright (c) 2014 Sean Doyle
+Copyright (c) 2015 Sean Doyle and thoughtbot, inc.
```
Comment (teoljungberg) [inline LICENSE.txt]: Why two copyrights? Can they be merged into one?

Comment (mike-burns) [inline LICENSE.txt]: One is for 2014 and one is for 2015, and they're different. How would you merge this?

Comment (teoljungberg) [inline LICENSE.txt]: I don't speak Licenses, but I would go with `Copyright (c) 2014 - 2015 Sean Doyle and thoughtbot, inc`

File: json-matchers.gemspec (hunk 3 above)
Comment (seanpdoyle) [inline json-matchers.gemspec]: Yes! ([from this comment](https://github.com/thoughtbot/json-matchers/pull/15#issuecomment-142948993))

**Comment by teoljungberg** [comment]
Go for it!

**Pull request merged**